pkg/loghttp/push/otlp.go

```diff
@@ -197,6 +199,7 @@ func otlpToLokiPushRequest(ld plog.Logs, userID string, tenantsRetention Tenants
 
 			scopeAttributesAsStructuredMetadataSize := labelsSize(scopeAttributesAsStructuredMetadata)
 			stats.StructuredMetadataBytes[tenantsRetention.RetentionPeriodFor(userID, lbs)] += int64(scopeAttributesAsStructuredMetadataSize)
+			stats.ResourceAndSourceMetadataLabels[tenantsRetention.RetentionPeriodFor(userID, lbs)] = append(stats.ResourceAndSourceMetadataLabels[tenantsRetention.RetentionPeriodFor(userID, lbs)], scopeAttributesAsStructuredMetadata...)
```

Comment:
Same here, this value should be unchanged, right? :thinking: 

```suggestion
			stats.StructuredMetadataBytes[retentionPeriodForUser] += int64(scopeAttributesAsStructuredMetadataSize)
			stats.ResourceAndSourceMetadataLabels[retentionPeriodForUser] = append(stats.ResourceAndSourceMetadataLabels[retentionPeriodForUser], scopeAttributesAsStructuredMetadata...)
```